team-validator.js

```diff
@@ -202,6 +178,19 @@ if (!process.send) {
 	});
 }
 
+/**
+ * Validates a username or Pokemon nickname
+ */
+
+var bannedNameStartChars = {'~':1, '&':1, '@':1, '%':1, '+':1, '-':1, '!':1, '?':1, '#':1, ' ':1};
+var toName = function (name) {
```

Comment:
Updated. Also switched to function declaration because I can (``toName`` used to be the result of an IIFE).